src/kbc_modules/offline_fs_kbc/common.rs

```diff
@@ -50,10 +51,25 @@ pub fn load_keys(keyfile_name: &str) -> Result<Keys> {
         .collect()
 }
 
+pub fn load_resources(resources_file_name: &str) -> Result<Resources> {
+    let resources_json = fs::read_to_string(resources_file_name)?;
+    let encoded_resources: HashMap<String, String> = serde_json::from_str(&resources_json)
+        .map_err(|_| anyhow!("Failed to parse resources JSON file"))?;
+    encoded_resources
+        .iter()
+        .map(|(k, v)| match decode(v) {
+            Ok(resource) => Ok((k.clone(), resource)),
+            Err(_) => Err(anyhow!("Failed to decode resource")),
+        })
+        .collect()
+}
+
 pub mod tests {
     pub use super::*;
-
-    pub use base64::encode;
+    use crate::kbc_modules::ResourceName::GPGPublicKey;
+    use crate::kbc_modules::ResourceName::Policy;
+    use crate::kbc_modules::ResourceName::SigstoreConfig;
```

Comment:
I was thinking something slightly different with this import. 

How about `use crate::kbc_modules::ResourceName`

and then when you use Policy below you would do `ResourceName::Policy`

Currently we have something like `Policy.to_string()`, which I think is a bit ambiguous, especially given all the different meanings of Policy across the project.

Reply:
updated